strawberry/http/__init__.py

```diff
@@ -33,6 +33,7 @@ class GraphQLRequestData:
     query: Optional[str]
     variables: Optional[Dict[str, Any]]
     operation_name: Optional[str]
+    extensions: Optional[Dict[str, Any]]
```

Comment:
Wondering if this should default to `None`?

```suggestion
    extensions: Optional[Dict[str, Any]] = None
```

Reply:
@bellini666 Since the `variables` parameter in the same object is also optional, and didn't have a default value, I wanted to keep this consistent so I don't introduce unexpected behaviour in case it gets initialised automatically some other way? 🤷‍♂️

Reply:
Fair point!

Maybe all of them should? =P
I'm fine in keeping the consistency or changing everything, lets see what others think